Article/Form.js

```diff
@@ -0,0 +1,72 @@
+//grab form
+let form = document.querySelector('form');
+console.log(form);
+
+// click event handler for submit button
+form.addEventListener('submit', event => {
+    event.preventDefault(); // to prevent form from auto-refreshing
+
+    // grab the values from the forms inputs
+    let newHeader = form.querySelector('#user-header').value; // input
+    let newContent = form.querySelector('#user-article').value; // textarea
+
+    //create new user article - div.article
+    let userArticle = document.createElement('div');
+    userArticle.className = 'article';
+
+    //h2
+    let userHeader = document.createElement('h2');
+    userHeader.textContent = newHeader;
+    userArticle.appendChild(userHeader);
+
+    //p.date
+    let date = document.createElement('p');
+    date.className = 'date';
+    //function to parse the date
+    function formatDate(date) {
+        var monthNames = [
```

Comment:
my only comment is please try to use const in these, try to get away from var completely in your code.